articles/connections/database/custom-db/templates/verify.md

```diff
@@ -26,8 +26,8 @@ When working on your user creation script, keep in mind that:
 Auth0 provides sample scripts for use with the following languages/technologies:
 
 * [JavaScript](#javascript)
-* [ASP.NET Membership Provider (MVC3 - Universal Providers)](#aspnet-membership-provider-mvc3---universal-providers)
-* [ASP.NET Membership Provider (MVC4 - Simple Membership)](#aspnet-membership-provider-mvc4---simple-membership)
+* [ASP.NET Membership Provider (MVC3 - Universal Providers)](#asp-net-membership-provider-mvc3-universal-providers-)
```

Comment:
@mpaktiti seems that those new links are broken again, is there a reason for that?

Reply:
Hey @aarighi They work for me. Maybe the fix wasn't deployed on the live environment when you checked. Check again?
https://auth0.com/docs/connections/database/custom-db/templates/verify#asp-net-membership-provider-mvc3-universal-providers-
https://auth0.com/docs/connections/database/custom-db/templates/verify#asp-net-membership-provider-mvc4-simple-membership-

Thanks!

Reply:
Sorry, both links work (: I was checking only the .md file

Reply:
No worries :) Thanks for helping in fixing this! 🙌